skins/oasis/css/core/PageHeader.scss

```diff
@@ -58,228 +60,39 @@
 		}
 	}
 
-	.view-source {
-		font-size: 10px;
-		position: relative;
-		top: -2px;
-
-		img {
-			padding: 0 3px;
-			vertical-align: baseline;
-		}
-
-		.sprite {
-			position: relative;
-			top: 1px;
-		}
-	}
-
-	h2,
-	p {
-		border: 0;
-		font-size: 14px;
-		margin-top: 4px;
-	}
-
-	.subtitle {
-		line-height: 1.5em;
-		margin: 1em 0;
-	}
-
-	img {
-		vertical-align: text-bottom;
-	}
-
+	.button,
 	.wikia-button,
 	.wikia-menu-button {
-		margin-top: 2px;
+		margin-top: 8px;
```

Comment:
As I mentioned above, it'll break depending on screen width and does need fixing before merging.